tests/testthat/helper-expect.R

```diff
@@ -173,12 +194,19 @@ expect_feedback <- function(
 }
 
 
-create_learnr_env <- function(user_code, solution_code = NULL, envir_prep, eval = TRUE) {
+create_learnr_env <- function(
+  user_code,
+  solution_code = NULL,
+  solution_code_all = NULL,
+  envir_prep,
+  eval = TRUE
+) {
```

Comment:
Not that we need to do it now, but it'd be nice to refactor these tests now that we have `mock_this_exercise()` and `learnr:::mock_exercise()`.